src/Range.ts

```diff
@@ -33,18 +33,30 @@ export class Range {
       new Position(endLine, endCharacter)
     )
   }
+
   public static fromNode(node: ts.Node): Range {
-    const sourceFile = node.getSourceFile()
+    // When getting a range of a node a special case is needed for
+    // the constructor case because it does not have a name node.
     const rangeNode: ts.Node = ts.isConstructorDeclaration(node)
       ? node.getFirstToken() ?? node
       : node
-    const start = sourceFile.getLineAndCharacterOfPosition(rangeNode.getStart())
-    const end = sourceFile.getLineAndCharacterOfPosition(rangeNode.getEnd())
+    return Range.getRangeFromNode(rangeNode)
+  }
+
+  public static fromNodeForEnclosing(node: ts.Node): Range {
```

Comment:
Any reason to keep this method when it's equivalent to `getRangeFromNode`?

Reply:
Thanks for the comment, it was confusing and I refactored to simplify this.